gas_stats.json

```diff
@@ -1,304 +1,326 @@
+        "cold": 570205,
+        "warm": 512479
     },
     "batchAction.borrowWithdrawUnderlying.NoDeposit": {
-        "cold": 405520,
-        "warm": 375306
+        "cold": 403447,
+        "warm": 373440
     },
     "batchAction.lend.DepositAsset": {
-        "cold": 292163,
-        "warm": 247221
+        "cold": 292396,
+        "warm": 247644
     },
     "batchAction.lend.DepositUnderlying": {
-        "cold": 397050,
-        "warm": 352106
+        "cold": 397192,
+        "warm": 352497
     },
     "batchAction.lend.NoDeposit": {
-        "cold": 211961,
-        "warm": 181344
+        "cold": 212374,
+        "warm": 182104
     },
     "batchAction.lend.RollToMaturity": {
-        "cold": 305965,
-        "warm": 305965
+        "cold": 310372,
+        "warm": 310372
     },
     "batchAction.lend.WithdrawToAsset": {
-        "cold": 211514,
-        "warm": 211514
+        "cold": 211908,
+        "warm": 211908
     },
     "batchAction.lend.WithdrawToUnderlying": {
-        "cold": 251458,
-        "warm": 251458
+        "cold": 251756,
+        "warm": 251756
     },
     "batchAction.liquidity.DepositAsset": {
-        "cold": 310225,
-        "warm": 267818
+        "cold": 307197,
+        "warm": 264859
     },
     "batchAction.liquidity.DepositUnderlying": {
-        "cold": 359016,
-        "warm": 316609
+        "cold": 356028,
+        "warm": 313620
     },
     "batchAction.liquidity.NoDeposit": {
-        "cold": 231309,
-        "warm": 231309
+        "cold": 228859,
+        "warm": 228859
     },
     "batchAction.liquidity.RollToMaturity": {
-        "cold": 255787,
-        "warm": 255787
+        "cold": 250658,
+        "warm": 250658
     },
     "batchAction.liquidity.WithdrawToAsset": {
-        "cold": 193858,
-        "warm": 193858
+        "cold": 192521,
+        "warm": 192521
     },
     "batchAction.liquidity.WithdrawToUnderlying": {
-        "cold": 233815,
-        "warm": 233815
+        "cold": 232314,
+        "warm": 232314
     },
     "deposit.asset": {
-        "cold": 158296,
-        "warm": 110513
+        "cold": 156907,
+        "warm": 109201
     },
     "deposit.underlying": {
-        "cold": 237590,
-        "warm": 189808
+        "cold": 236364,
+        "warm": 188658
     },
     "deposit.underlying.hasTransferFee": {
-        "cold": 245200,
-        "warm": 197418
+        "cold": 243973,
+        "warm": 196268
+    },
+    "fcashCrossCurrencyLiquidate.fCashAssets": {
+        "cold": 426277,
+        "warm": 426277
+    },
+    "liquidateCollateralCurrency.transferCash": {
+        "cold": 329511,
+        "warm": 329511
+    },
+    "liquidateCollateralCurrency.transferNTokens": null,
+    "liquidateCollateralCurrency.withdrawTokens": {
+        "cold": 368316,
+        "warm": 368316
+    },
+    "liquidateLocalCurrency.transferCash": {
+        "cold": 361778,
+        "warm": 361778
+    },
+    "liquidatefCashCrossCurrency.fCashAssets": null,
```

Comment:
Looks like this one is a duplicate of the key above.

Reply:
fixed